app/controllers/recordings_controller.rb

```diff
@@ -0,0 +1,53 @@
+class RecordingsController < ApplicationController
+  def index
+    @q = Recording.ransack(params[:q])
+    @recordings = @q.result(distinct: true)
+    render 'welcome/index'
+  end
+
+
+  def edit
+  end
+
+  def update
+  end
+
+  def new
+    @recording = Recording.new
+  end
+
+  def create
+    @recording = Recording.new(recording_params)
+    if  @recording.save
+      redirect_to recording_path(@recording)
+    else
+      "ERROR"
+      # make validations
+    end
+  end
+
+  def show
+    @recording = Recording.find(params[:id])
+  end
+
+  def delete
+  end
+
+  def like
+    @recording = Recording.find(params[:id])
+    @recording.liked_by current_user
```

Comment:
Using past-tense names suggests you are reading an attribute. make_like_from or similar would be better - use active verb forms for things that do something.